example/package.json

```diff
@@ -12,55 +12,56 @@
     "@giphy/react-native-sdk": "2.1.3",
     "@likeminds.community/chat-rn": "1.5.3",
-    "@likeminds.community/chat-rn-core": "0.5.10",
-    "@notifee/react-native": "7.6.1",
-    "@react-native-async-storage/async-storage": "^1.18.1",
+    "@likeminds.community/chat-rn-core": "file:../likeminds-chat-reactnative-integration/likeminds.community-chat-rn-core-0.5.10.tgz",
     "@react-native-clipboard/clipboard": "1.11.2",
     "@react-native-community/datetimepicker": "7.4.1",
     "@react-native-community/slider": "4.4.3",
-    "@react-native-firebase/app": "18.1.0",
-    "@react-native-firebase/messaging": "18.1.0",
-    "@react-native/babel-preset": "0.74.0",
-    "@react-navigation/material-top-tabs": "6.6.2",
-    "@react-navigation/native": "6.1.2",
-    "@react-navigation/native-stack": "6.9.8",
-    "@react-navigation/stack": "6.3.29",
-    "@shopify/flash-list": "1.4.3",
     "@types/react-native-video": "5.0.19",
-    "aws-sdk": "2.1380.0",
-    "buffer": "6.0.3",
-    "diff": "5.1.0",
-    "firebase": "9.17.1",
-    "lottie-react-native": "6.4.0",
-    "moment": "2.29.4",
     "react": "18.2.0",
     "react-native": "0.71.1",
     "react-native-audio-recorder-player": "3.6.4",
     "react-native-blob-util": "0.19.2",
     "react-native-compressor": "1.6.1",
     "react-native-create-thumbnail": "1.6.4",
     "react-native-device-info": "10.4.0",
-    "react-native-document-picker": "8.2.0",
-    "react-native-gesture-handler": "2.9.0",
     "react-native-image-crop-tools": "1.6.4",
     "react-native-image-picker": "7.0.2",
     "react-native-linear-gradient": "2.8.3",
     "react-native-media-console": "2.0.6",
     "react-native-pager-view": "6.1.4",
     "react-native-pdf-thumbnail": "1.2.1",
-    "react-native-permissions": "3.10.1",
-    "react-native-reanimated": "3.4.2",
     "react-native-safe-area-context": "4.5.0",
-    "react-native-screens": "3.19.0",
-    "react-native-shimmer-placeholder": "2.0.9",
     "react-native-svg": "13.9.0",
+    "react-native-permissions": "3.10.1",
+    "react-native-screens": "3.19.0",
+
     "react-native-swiper-flatlist": "3.2.2",
-    "react-native-tab-view": "3.5.0",
     "react-native-track-player": "4.0.1",
     "react-native-uuid": "2.0.1",
     "react-native-video": "5.2.1",
     "realm": "11.10.2",
-    "rn-emoji-keyboard": "1.2.1"
+    "rn-emoji-keyboard": "1.2.1",
+    "react-native-document-picker": "8.2.0",
+    "react-native-gesture-handler": "2.9.0",
+    "react-native-reanimated": "3.4.2",
+    "@react-navigation/material-top-tabs": "6.6.2",
+    "@react-navigation/native": "6.1.2",
+    "@react-navigation/native-stack": "6.9.8",
+    "@react-native-firebase/app": "18.1.0",
+    "@react-native-firebase/messaging": "18.1.0",
+    "firebase": "9.17.1",
+    "lottie-react-native": "6.4.0",
+    "@shopify/flash-list": "1.4.3",
+    "aws-sdk": "2.1380.0",
+    "buffer": "6.0.3",
+    "diff": "5.1.0",
+    "moment": "2.29.4",
+    "@notifee/react-native": "7.6.1",
```

Comment:
this `package.json` is a bit different then what you have shared, pls check once